pilot/docker/envoy_telemetry.yaml.tmpl

```diff
@@ -1,262 +0,0 @@
-admin:
```

Comment:
Where is this file now?

Reply:
https://github.com/istio/istio/blob/6fab76ef7ba34a9e551e0313b67371064cc3a522/manifests/istio-telemetry/mixer-telemetry/templates/configmap-envoy.yaml#L5

Reply:
I'm not sure how you can get internal Mixer telemetry without using a custom roller bootstrap.

Reply:
We still have a custom bootstrap

Reply:
I see. Although I'm not sure I agree to place it in the installation. I think mixer codebase would be the appropriate place for its bootstrap.

Reply:
In general, installation files have extremely poor coverage since each test ends up being e2e. If we can separate resources away from k8s/install, then we can test them locally in a unit test without running istio or even k8s.

Reply:
I don't necessarily disagree but this is completely unrelated to the PR, can you open an issue if you want to change this?

Reply:
Well, I'm not the one moving them :) I'd rather leave things the way they were even if it completely makes no sense (which I agree with). Otherwise, we have to make them better, and it's not worth it.